Makefile

```diff
@@ -263,6 +341,7 @@ install: dist
 	install -m 0644 README.md $(PREFIX)/$(DOCDIR)/scap-security-guide
 	@# install a symlink in the old content location for compatibility
 	install -d $(PREFIX)/$(DATADIR)/xml/scap/ssg
-	ln -sf $(PREFIX)/$(DATADIR)/scap/ssg $(PREFIX)/$(DATADIR)/xml/scap/ssg/content
+	ln -sf ../../../scap/ssg $(PREFIX)/$(DATADIR)/xml/scap/ssg/content
```

Comment:
@mpreisler why not keep `$(PREFIX)/$(DATADIR)`? `../../../` seems like a poor replacement that could be error prone.